catalog/spatial/registry/registry-schema-bindings/src/test/java/org/codice/ddf/registry/schemabindings/helper/RegistryPackageTypeHelperTest.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
+import java.util.Locale;
```

Comment:
This import is in the wrong order. It should be moved up one line so it is between `List` and `Map` (alphabetically sorted).

Once that is resolved, this PR can be merged.